contrib/vagrant/provision-util.sh

```diff
@@ -14,8 +14,8 @@ os::provision::build-origin() {
 
   # This optimization is intended for devcluster use so hard-coding the
   # arch in the path should be ok.
-  if [[ -f "$(os::build::find-binary oc "${origin_root}")" &&
-          "${skip_build}" = "true" ]]; then
+  if ! OS_ROOT="${origin_root}" os::util::find::built_binary oc >/dev/null 2>&1 &&
```

Comment:
In the old logic, we skip the build if the binary exists and we are told to skip the build.  In the new logic, we skip the build if the binary does *not* exist and we are told to skip the build.  Do I misunderstand? Did the logic get inverted from what it is supposed to be?